nomenclature/definitions/variable/energy/energy-services.yaml

```
@@ -0,0 +1,23 @@
+# List of variables related to energy services 
+
+# The tag <Transport> can be replaced by any item in the list in `transport_types.yaml`
+
+Energy Service|Transportation:
+   description: Annual provision of energy services related to all transportation modes
```

Please remove "Annual" (also in subsequent definitions) - this would be clarified in the template via the "Subannual" column - if there is no subannual entry, it is annual data, otherwise it is whatever that column specifies

```suggestion
   description: Provision of energy services related to all transportation modes
```